docs/AVM_Specification.md

```diff
@@ -118,19 +118,14 @@ The implementation of the Runtime Environment will vary in different scenarios.
 The runtime environment will supply values for:
 
 - lower and upper bounds on the time (Integers)
-- a Value that is the contents of VM’s Inbox
+- a queue of message values representing the VM's inbox
```

Comment:
Updated. The idea was that removing messages is an dequeue operation and adding messages is a enqueue operation since the inbox is a FIFO structures as handled in the VM.

Removing the following would eliminate that addition operation. Do you think we should keep that or get rid of it.

> Messages may arrive in the Inbox at any time (but not during the execution of an inbox instruction). Any newly arriving messages will be appended after the messages that were previously in the Inbox.

Reply:
Oh, this is in the spec.  I feel better about calling it a queue in the spec.